cmd/medius/docs/docs.go

```diff
@@ -60,10 +60,12 @@ func run(options *common.Options) error {
 		}
 
 		focusMatched = true
-		log := common.Logger(p.Artifact)
-		name := p.Artifact.Metadata().Name
 
-		description, err := createDescription(p.Artifact, options.PublishDocsOptions.Registry)
+		artifact := p.Artifacts[0]
```

Comment:
What about other artifacts in slice?

Reply:
The difference between artifacts is only architecture. E.g. all CentOS artifacts will have the same description [1] so it's not needed to iterate over all of them, and it's enough to reference only one from the array.

[1] https://github.com/kubevirt/containerdisks/blob/main/artifacts/centos/centos.go#L19-L22